src/gui/search/searchjobwidget.cpp

```diff
@@ -211,11 +211,11 @@ void SearchJobWidget::cancelSearch()
     m_searchHandler->cancelSearch();
 }
 
-void SearchJobWidget::downloadTorrents()
+void SearchJobWidget::downloadTorrents(ShowAddNewTorrentDialog showDialog)
```

Comment:
`const ShowAddNewTorrentDialog showDialog` and all similar instances below. (only add this `const` in .cpp file)

Reply:
Fixed.

Reply:
In the feature, when should I add 'const' to function parameters passed by value? Should I do it, just like in case of local variables, whenever it makes sense to emphasize the fact that value don't change? Should I do it for bool's, int's etc.?

Reply:
>Should I do it, just like in case of local variables, whenever it makes sense to emphasize the fact that value doesn't change? 

Yes.

>Always if possible? 

Yes.

>I'm a bit confused since in current qB source code function parameters passed by value are mostly non-const...

Legacy code remnants... You should follow the latest coding guidelines.